.vscode/launch.json

```diff
@@ -13,6 +13,23 @@
       "request": "attach",
       "processId": "${command:PickProcess}"
     },
+    {
+      "name": "Debug Current Vitest File",
+      "presentation": {
+        "group": "__TOP__"
+      },
+      "type": "node",
+      "autoAttachChildProcesses": true,
+      "skipFiles": ["<node_internals>/**", "**/node_modules/**"],
+      "program": "${workspaceFolder}/common/temp/node_modules/vitest/vitest.mjs",
+      "args": [
+        "run",
+        "${relativeFile}",
+        "--no-file-parallelism"
+      ],
+      "smartStep": true,
```

Comment:
what does smartStep do?

Reply:
It helps when running through transpiled .js code - some parts of the code aren't covered by source maps, like async/await processes. We would frequently hit these weird parts as we debug, normally you'd have to 'step out' until you go back to a part of the code that is resolved by source maps. `smartStep` skips showing you those parts automatically for you. [Reference](https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_smart-stepping)